index.html

```diff
@@ -25,62 +25,62 @@
   <header class="menu-bar">
     <nav>
       <div>
-        <a href="index.html" id="logo">Dasik Band</a>
+        <a href="index.html" id="logo"><img src="./images/dasik-logo.svg" alt="Dasik Band"></a>
         <button id="hamburger" type="menu"><i class="fa-solid fa-bars"></i></button>
       </div>
       <ul id="menu-links">
         <li class="menu-link"><a href="index.html">Home</a></li>
         <li class="menu-link"><a href="about.html">About</a></li>
-        <li class="menu-link"><a href="mailto:dasikband@gmail.com"><i class="fa-solid fa-envelope"></i> Contact Us</a></li>
+        <li class="menu-link"><a href="mailto:dasikband@gmail.com"><i class="fa-solid fa-envelope"></i> Contact</a></li>
       </ul>
     </nav>
   </header>
   <section id="intro">
     <div class="wrapper">
-      <h2>"Hello! Sharing World"</h2>
-      <h1 class="clip-letters">CREATIVE COMMONS GLOBAL SUBMIT 2015</h1>
-      <div class="phrase">A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in October, in Korea</div>
-      <h3>2015.10.15(THU) ~ 16(FRI)</h3>
-      <p>@National Museum of Korea, Art Center Nabi and more</p>
+      <h2>"Hello! Introducing to the world!"</h2>
+      <h1 class="clip-letters">DASIK BAND EVENTS PAGE 2022</h1>
+      <div class="phrase">Two brothers who decided to create our own sound, thus becoming a new Ecuadorian musical act with a non-mainstream approach.</div>
+      <h3>2022.05.19(THU) ~ 20(FRI)</h3>
+      <p>@Casa de la Cultura Ecuatoriana (Ecuadorian House of Culture).</p>
     </div>
   </section>
   <section id="program">
     <div class="wrapper">
-      <h2>Main program</h2>
+      <h2>Multimedia</h2>
       <hr>
       <ul>
         <li>
-          <img src="./images/lecture.svg" alt="Lecture">
-          <h3>Lecture</h3>
-          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nemo, vitae?</p>
+          <i class="fa-solid fa-music"></i>
+          <h3>Songs</h3>
+          <p>Temblando <br>Tu piel</p>
         </li>
         <li>
-          <img src="./images/lecture.svg" alt="Lecture">
-          <h3>Lecture</h3>
-          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nemo, vitae?</p>
+          <i class="fa-solid fa-video"></i>
+          <h3>Video</h3>
+          <p>La Aldea Pub Concert<br>Katari Gig</p>
```

Comment:
- Good job in this program section but it still needs a few finishing touches 😄 
- [ ] First, your program section is missing a button underneath it in the mobile version and an underlined text in the desktop version. Kindly add those.
- [ ] Also, it would be great if you could align the texts as in the original design. Right now your text alignment is out of sync.
- [ ] Next, in the desktop view, you could add the white border hover effect as shown in the original design. This will improve the UI of your site.
- Your desktop design:
![Annotation 2022-05-19 040501](https://user-images.githubusercontent.com/85113567/169197786-af9943b8-077d-4689-b7cf-26c69ba8cde8.jpg)
- Required desktop design:
![Annotation 2022-05-19 040447](https://user-images.githubusercontent.com/85113567/169197824-a759e5d1-d5d5-4686-b4e6-15aaa6420578.jpg)

- Your mobile design:
![Annotation 2022-05-19 040605](https://user-images.githubusercontent.com/85113567/169197859-8e7af0e1-d886-4602-ae4f-9d40f997a8d6.jpg)
- Required mobile design:
![Annotation 2022-05-19 040621](https://user-images.githubusercontent.com/85113567/169197880-3e8581bb-50e7-4a32-9e47-4affdea4dd21.jpg)